test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -486,6 +583,13 @@ public String ipAddress(int nodeNum)
             {
                 throw new UncheckedIOException(e);
             }
+            cluster.setUncaughtExceptionsFilter((node, t) -> {
+                // api is "ignore" so false means include,
+                var rootCause = Throwables.getRootCause(t);
+                if (rootCause.getMessage() != null && rootCause.getMessage().startsWith("Queried for epoch") && rootCause.getMessage().contains("but could not catch up. Current epoch:"))
```

Comment:
honestly I only did this as it doesn't *seem* to impact the test, but it did cause it to fail as unexpected... I can remove if you feel this *should* fail the test...